apps/web-portal-client/lib/components/LoginButton.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export function LoginButton(props) {
+  const {loginStatus, setLoginStatus} = props
```

Comment:
it doesn't make sense to pass 'setLoginStatus' here from parent

Choose one of : 

1. Creat state here
1. Keep state on parent and pass here only proper text

Reply:
How can I change loginStatus inside LoginButton without passin setLoginStatus function if I decided to left this state in parents component?

Reply:
Yes, realy its not obligatiry to pass loginStatus - I can make checking status in parents componetn and pass only result for rendering, but if smb clicks on this button, status must change so I need to pass some function for onclick in btn. may by it is good way not to extand this component out?

Reply:
Or pass just onClick function as part of interface. Somethig like this
```suggestion
  <LoginButton
              label={loginStatus ? 'Logout' : 'Login'}
              onClick={() => setLoginStatus(!loginStatus)}
            />
            <MobileMenuButton
              shape={menuStatus ? 'M6 18L18 6M6 6l12 12' : 'M4 6h16M4 12h16M4 18h16'}
              onClick={() => setMenuStatus(!menuStatus)}
            />
```

Reply:
Yep, something like this

Reply:
Usually we try to avoid components to have a state, so if we need to inform we introduce onSomething property